<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Allen Winter &lt;winter@kde.org&gt;</author>
 <comment>Configuration page for the Korganizer Special Dates Summary</comment>
 <class>SDSummaryConfig_Base</class>
 <widget class="QWidget" name="SDSummaryConfig_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Dates Summary Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="mDaysGroup">
     <property name="title">
      <string>Show Special Dates Starting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="mDateTodayButton">
        <property name="toolTip">
         <string>Show special occasions for today only</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to see special occasions occurring on this date only.</string>
        </property>
        <property name="text">
         <string>&amp;Today only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mDateMonthButton">
        <property name="toolTip">
         <string>Show special occasions starting within the next month</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to see special occasions that start sometime during the next 31 days.</string>
        </property>
        <property name="text">
         <string>Within the next &amp;month (31 days)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="mDateRangeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Select the days for showing upcoming special occasions</string>
          </property>
          <property name="whatsThis">
           <string>Check this box if you want to specify the number of days in the future for upcoming special occasions.</string>
          </property>
          <property name="text">
           <string>Within the &amp;next:</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mCustomDays">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Set the number of days to show upcoming special occasions</string>
          </property>
          <property name="whatsThis">
           <string>Use this spinbox to set the number of days to show upcoming special occasions up to one year in the future.</string>
          </property>
          <property name="specialValueText">
           <string>1 day</string>
          </property>
          <property name="suffix">
           <string comment="days to show in summary"> days</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mShowFromCalGroup">
     <property name="title">
      <string>Show These Special Dates From Your Calendar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="mShowBirthdaysFromCalBox">
        <property name="toolTip">
         <string>Show birthdays from your calendar</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include birthdays from your calendar in the upcoming special occasions summary. Typically, these are events with the &quot;Birthday&quot; category.</string>
        </property>
        <property name="text">
         <string>Show &amp;birthdays</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowAnniversariesFromCalBox">
        <property name="toolTip">
         <string>Show anniversaries from your calendar</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include anniversaries from your calendar in the upcoming special occasions summary. Typically, these are events with the &quot;Anniversary&quot; category.</string>
        </property>
        <property name="text">
         <string>Show &amp;anniversaries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowHolidaysFromCalBox">
        <property name="toolTip">
         <string>Show holidays from your calendar</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include holidays from your calendar in the upcoming special occasions summary. These are events from your configured holiday region, or events with the &quot;Holiday&quot; category.</string>
        </property>
        <property name="text">
         <string>Show &amp;holidays</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowSpecialsFromCalBox">
        <property name="toolTip">
         <string>Show special occasions from your calendar</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include events from your calendar with the &quot;special occasion&quot; category.</string>
        </property>
        <property name="text">
         <string>Show s&amp;pecial occasions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mShowFromKABButtonGroup">
     <property name="title">
      <string>Show These Special Dates From Your Contact List</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="mShowBirthdaysFromKABBox">
        <property name="toolTip">
         <string>Show birthdays from your address book</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include birthdays from your address book in the upcoming special occasions summary.</string>
        </property>
        <property name="text">
         <string>Show birth&amp;days</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowAnniversariesFromKABBox">
        <property name="toolTip">
         <string>Show anniversaries from your address book</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include anniversaries from your address book in the upcoming special occasions summary.</string>
        </property>
        <property name="text">
         <string>Show anni&amp;versaries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Groupware Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="mShowMineOnly">
        <property name="toolTip">
         <string>Show events belonging to my calendars only</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want the summary to show events belonging to your calendars only.</string>
        </property>
        <property name="text">
         <string>Show &amp;my events only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mDateRangeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mCustomDays</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
